### PR TITLE
[coq] Adapt to coq/coq#11836

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

**Comment by ejgallego** [comment]
Please merge.

File: src/Rewriter/Util/plugins/definition_by_tactic.ml.v813 [hunk 1]
```diff
@@ -5,7 +5,8 @@ let make_definition_by_tactic sigma ~poly (name : Names.Id.t) (ty : EConstr.t) (
   let info = Declare.Info.make ~poly () in
   let lemma = Declare.Proof.start ~cinfo ~info sigma in
   let lemma, _ = Declare.Proof.by tac lemma in
-  let ids = Declare.Proof.save ~proof:lemma ~opaque:Vernacexpr.Transparent ~idopt:None in
+  let pm = Declare.OblState.empty in
+  let _pm, ids = Declare.Proof.save ~pm ~proof:lemma ~opaque:Vernacexpr.Transparent ~idopt:None in
```
Comment (SkySkimmer) [inline src/Rewriter/Util/plugins/definition_by_tactic.ml.v813:9]: We have save_regular now btw

Comment (ejgallego) [reply src/Rewriter/Util/plugins/definition_by_tactic.ml.v813]: I was lazy but in the end, I updated it.

**Comment by JasonGross** [comment]
I'll merge once the CI completes.  (I should really update the master job to pull from the latest git version / use opam, so I don't need to wait for Launchpad to rebuild and publish the master package before the CI can pass...)

**Comment by JasonGross** [comment]
Merged, thanks!

**Pull request merged**